scripts/load_testing/create_account.py

```diff
@@ -35,7 +35,7 @@ def signup(self):
         link = dom.find("a[href*='confirmation_token']")[0].attrib['href']
 
         # click email confirmation link and submit password
-        resp = self.client.get(link, auth=auth)
+        resp = self.client.get(link, auth=auth, name='/sign_up/email/confirm?confirmation_token=')
```

Comment:
I think the issue here is that link is not what you expect. Try this syntax on L35:

```
link = dom.find("a[href*='confirmation_token']").eq(0).attr('href')
```
I think that may get you what you want.

Reply:
I believe the link is correct, I am able to create new accounts. The problem is with how the results are tallied. We need to group the request under one name space so we can make sense of the results.

before: 
``` GET /sign_up/email/confirm?confirmation_token=UB8civbYexHgDYUzkH3z       1     0(0.00%)     719     719     719  |     720    0.00
 GET /sign_up/email/confirm?confirmation_token=enbcpKBnpdNJC-7fdVZL       1     0(0.00%)     685     685     685  |     680    0.00
 GET /sign_up/email/confirm?confirmation_token=xAgs5LPpCmk4kuLKG9u7       0     0(0.00%)       0       0       0  |       0    0.00
 GET /sign_up/email/confirm?confirmation_token=z79prkV9JwqZ3afy6oj_       1     0(0.00%)     705     705     705  |     700    0.00
```

after: 
```
Percentage of the requests completed within given times
 Name                                                           # reqs    50%    66%    75%    80%    90%    95%    98%    99%   100%
--------------------------------------------------------------------------------------------------------------------------------------------
 GET /                                                               2    140    140    140    140    140    140    140    140    139
 GET /api/saml/logout                                                3    860    860   1200   1200   1200   1200   1200   1200   1246
 POST /login/two_factor/sms                                          3   1500   1500   1600   1600   1600   1600   1600   1600   1610
 POST /phone_setup                                                   3   2400   2400   2700   2700   2700   2700   2700   2700   2683
 GET /profile                                                        3    880    880   1300   1300   1300   1300   1300   1300   1324
 POST /sign_up/create_password                                       2   1400   1400   1400   1400   1400   1400   1400   1400   1429
 GET /sign_up/email/confirm?confirmation_token=                      3    640    640   1100   1100   1100   1100   1100   1100   1052
 GET /sign_up/enter_email                                            2    410    410    410    410    410    410    410    410    408
 POST /sign_up/enter_email                                           3    980    980   1100   1100   1100   1100   1100   1100   1147
 POST /sign_up/personal_key                                          3   1300   1300   1500   1500   1500   1500   1500   1500   1519
--------------------------------------------------------------------------------------------------------------------------------------------
```

Note how the `GET /sign_up/email/confirm?confirmation_token=`'s are grouped.

Reply:
I think -- and I definitely could be wrong here -- that you're seeing confirmation link be either an empty string or "None". I'm having trouble confirming, because I'm throwing an index error on the [0] every time.  From what I'm seeing in the pyquery docs, the syntax should be `link = dom.find("a[href*='confirmation_token']").eq(0).attr('href')`. the difference in how it gets first and attrs makes me wonder if we may have different versions of pyquery installed. 

we should be able to check the confirmation_token by throwing in a pdb to find out for sure, but like I said, I haven't been able to due to the index error.  I think we should solidify our confirmation token getter, then make the other changes here if we still need to.

Hang on though, I'm gonna catch the indexError and inspect more.

Reply:
The change made here doesn't change the GET request for the confirmation token to the server it only changes how it's reported in locust's progress report (shown above). I know the confirmation link is working on as expected as it's able to setup and register the users and make it to the profile page.  

After chatting on slack, you were correct there's an issue when using the two different versions of pyquery. We'll create a new PR to force v1.3.